src/Install/Steps/StoreConfig.php

```diff
@@ -55,10 +55,13 @@ private function buildConfig()
     {
         return [
             'debug'    => $this->debugMode,
-            'poweredByHeader'  => true,
             'database' => $this->dbConfig->toArray(),
             'url'      => (string) $this->baseUrl,
             'paths'    => $this->getPathsConfig(),
+            'headers'  => [
+                'poweredByHeader'  => true,
```

Comment:
Noticed this as I scrolled past and it couldn't stop bugging me 🙈 

```suggestion
                'poweredByHeader' => true,
```